src/components/Sidebar/Sidebar.jsx

```diff
@@ -20,11 +41,46 @@ export const Sidebar = ({ children, className, footer, notifications, isFixed =
                 className,
             )}
         >
-            <div className={clsx("p-2 text-center xl:text-left", notifications ? null : "invisible")}>
-                <Counter className="text-sm">{notifications}</Counter>
+            <div className="flex p-2 sm:justify-center xl:justify-between">
+                <div
+                    className={clsx(
+                        "cursor-pointer sm:text-center",
+                        notifications?.announcements?.count ? null : "hidden",
+                    )}
+                >
+                    <Counter className="mr-2 text-sm" onClick={handleAnnounceClick} style={style}>
+                        <AnnounceIcon className="mr-1 sm:hidden xl:block" />
+                        {notifications?.announcements?.count}
+                    </Counter>
+                </div>
+
+                <div className={clsx("cursor-pointer sm:text-center", notifications?.notices?.count ? null : "hidden")}>
+                    <Counter className="text-sm" onClick={handleNotificationClick}>
+                        <BellIcon className="sm:hidden xl:block" />
+                        {notifications?.notices?.count}
+                    </Counter>
+                </div>
             </div>
 
-            <div className="mb-10 text-center">
+            <Drawer
+                classNames={{ left: "md:left-24 xl:left-50" }}
+                position="left"
+                title="Announcements"
+                content={notifications?.announcements?.content}
+                isOpen={isAnnouncementsOpen}
+                onClose={() => setIsAnnouncementsOpen(false)}
```

Comment:
Same here. Announcements and notifications are too specific. Make the var names generic. 

DM @nemanjakrstic He's better at naming things than me

Reply:
@rushi I don't have better names than "left" and "right" when architected like this.

This should have been a slot for the "header" in the sidebar. Then you can put whatever you want in the header of the sidebar.

```tsx
<Sidebar
    header={
        <div className="flex items-center justify-between">
            <Badge>1</Badge>
            <Badge>2</Badge>
        </div>
    }
/>
```

That way the sidebar doesn't not know about drawers, announcements, notifications, counts, clicks, anything.

Reply:
ok, we'll leave it as is, and deal with the renames in the refactor ticket.